recipes/r-biodb/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#export HOME=$(pwd)
+#mkdir "$HOME/.R"
+#cat >$HOME/.R/Makevars <<EOF
+#CXX=clang++
+#EOF
+echo %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+cat /proc/version
+echo %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+cat /etc/issue
+echo %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+env
+echo %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+alias
+echo %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
BTW: I used to just put a line `sh` in here (or `bash --norc`) to get "into" the environment and be able to poke around. Not sure if that still works with the docker - it does with `conda build` though.